doc/whats_new/v1.5.rst

```diff
@@ -43,6 +43,9 @@ Security
 Changed models
 --------------
 
+- |Efficiency| The `clustering.MeanShift` class has now improved computational speed as it properly converges for constant data.
+  :pr:`28951` by :user:`Akihiro Kuno <akikuno>`.
```

Comment:
```suggestion
- |Fix| The :class:`cluster.MeanShift` class now properly converges for constant data.
  :pr:`28951` by :user:`Akihiro Kuno <akikuno>`.
```

Reply:
Once this suggestion is applied, this entry of the changelog should subsequently be moved to the `sklearn.cluster` section of this page (if it already exists and create it if not).